app/assets/scripts/redux/store.js

```diff
@@ -3,13 +3,14 @@ import thunkMiddleware from 'redux-thunk';
 import { syncHistory } from 'react-router-redux';
 import reducer from './reducer';
 import { hashHistory } from 'react-router';
+import logger from 'redux-logger';
 
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
 export default createStore(
   reducer,
   composeEnhancers(
-    applyMiddleware(syncHistory(hashHistory), thunkMiddleware)
+    applyMiddleware(logger, syncHistory(hashHistory), thunkMiddleware)
```

Comment:
I feel like this should be a configuration variable at build time. @danielfdsilva can help here.

Reply:
Oh right, I forgot to take that out. I think there's a `process.env.DS_ENV === 'staging'` check common for picking up staging configs. I'll make the logger part of staging.